metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
I think documenting the Jira for follow up on that is fine.  That being said, I don't know if it is 'correct' with apache to cut a release without verification, and can be put off. @mattf-horton?